### PR TITLE
Webapp refactor

### PR DESCRIPTION
**Pull request by anisfeld**
Includes examples and new layout.

### CODE REVIEW

File: src/aequitas_webapp/views.py [hunk 1]
```diff
@@ -198,6 +198,6 @@ def report(dirname, name, reportid):
     )
 
 
-@app.route('/example.html', methods=['GET'])
+@app.route('/example', methods=['GET'])
```
Comment (jesteria) [inline src/aequitas_webapp/views.py]: any reason to strip this off of the URL path? the resource is just the read-only HTML page, after all. the site can use these or not, but a bunch of its other, similar resources identify themselves this way.

Comment (saleiro) [reply src/aequitas_webapp/views.py]: @anisfeld

Comment (anisfeld) [reply src/aequitas_webapp/views.py]: I don't know why I did that. I'll adjust it.

Comment (anisfeld) [reply src/aequitas_webapp/views.py]: Done.

Comment (jesteria) [reply src/aequitas_webapp/views.py]: not a big deal obviously, but cool. thanks :smile:

**Pull request merged**